api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1285,6 +1348,19 @@ type ClusterAutoscaling struct {
 	//
 	// +optional
 	PodPriorityThreshold *int32 `json:"podPriorityThreshold,omitempty"`
+
+	// expanders guide the autoscaler in choosing node groups during scale-out.
+	// Sets the order of expanders for scaling out node groups.
+	// Options include:
+	// * LeastWaste - selects the group with minimal idle CPU and memory after scaling.
+	// * Priority - selects the group with the highest user-defined priority.
+	// * Random - selects a group randomly.
+	// If not specified, `Random` is the default.
+	// Maximum of 3 expanders can be specified.
+	// +kubebuilder:validation:MaxItems=3
+	//
+	// +optional
+	Expanders []ExpanderString `json:"expanders,omitempty"`
```

Comment:
Should also set `MinItems` 1 to prevent an unstructured client from setting `expanders: []` which wouldn't round trip through a structured client